### ISSUE TITLE
Find graph creating tools for react

### ISSUE THREAD

**Issue by AntonOstman** [PUM04/pum-04#19]
**Is your feature request related to a problem? Please describe.**

Currently we have no tools for generating Graphs such as histograms. Creating home made graphs will take time and be hard

**Describe the solution you'd like**

We should have some suggestions of tools that are avaible for react that can create graphs such as histograms.

**Comment by AlgotE** [comment]
Available tools to look into:
[Recharts](https://recharts.org/)
[Victory](https://formidable.com/open-source/victory/)
[Visx](https://airbnb.io/visx/)
[Nivo](https://nivo.rocks/)

**Comment by AntonOstman** [comment]
NFC group is using TOAST ui so using that could be an advantage. However I think recharts looks good and seems to be what most people use. I will try creating some simple graphs with it to see how easy it is to use.

**Comment by AntonOstman** [comment]
This is what I have gathered so far
> [Recharts](https://recharts.org/) 
Seems to have good docs and is easy to use. However does not have good support for box diagrams, there is one example in the storybook but it is not build into the api. It also is well integrated with react.

> [Victory](https://formidable.com/open-source/victory/)  
Has what we need with good examples of how to get it running. However not as easy as recharts

> [Visx](https://airbnb.io/visx/)
Has tons of features but examples and docs are not so good which would probably make it hard to use.

> [Nivo](https://nivo.rocks/) 
Has good documentation and examples with where items can changed directly on their website before placing in our code. The only problem is I can not find a boxplot

> [Toast ui](https://ui.toast.com/) 
Is used by another group which could make for some good collaboration. Has everything we need, but implementation seems to be harder than for example with recharts.

I would personally chose recharts just because it is easy to use. boxplots could be a bit annoying to implement since there is no simple api call but there are 
examples of how to do it.
I attached a picture with what could be created in recharts in quite a short time
![image](https://user-images.githubusercontent.com/95719901/227194430-2d115878-dc2f-45e8-98f5-7777b5bbccb9.png)

And to create this graph with 2 bars this is all i needed to do was

```
import { Bar,BarChart, LineChart, Line, CartesianGrid, XAxis, YAxis } from 'recharts';

const data = [
  { name: 'Metric A', uv: 100, pv: 100, amt: 2400 },
  { name: 'Metric B', uv: 200, pv: 200, amt: 2200 },
  { name: 'Metric C', uv: 230, pv: 300, amt: 2200 },
  { name: 'Metric D', uv: 300, pv: 430, amt: 2500 },
  { name: 'Metric E', uv: 800, pv: 100, amt: 2400 },
];

const renderBarChart = (
  <BarChart width={500} height={500} data={data}>
    <CartesianGrid stroke="#ccc" />
    <Bar type="monotone" dataKey="uv" fill="#8884d8"/>
    <Bar type="monotone" dataKey="uv" fill="#82ca9d"/>
    <XAxis dataKey="name" />
    <YAxis />

  </BarChart>
);
```

And inside a div in the html to render the graph

```
{renderBarChart}
```

**Comment by AntonOstman** [comment]
I have now tested [Victory](https://formidable.com/open-source/victory/) aswell and it is quite nice. It was harder to use than recharts but has built in histogram and boxPlot. It also seems more powerful but I have not done super complicated things.

Here is a picture of what I was able to do during this morning
![image](https://user-images.githubusercontent.com/95719901/227510761-fd2f3dbf-35d5-4d8a-af10-ee27a41c92db.png)

If anyone is curious about the code it is in this branch 
> https://github.com/PUM04/pum-04/tree/19-find-graph-creating-tools
